spring-integration-ip/src/test/java/org/springframework/integration/ip/dsl/IpIntegrationTests.java

```diff
@@ -128,6 +128,13 @@ public void testUdp() {
 		assertEquals("foo", Transformers.objectToString().transform(received).getPayload());
 	}
 
+	@Test
+	public void testUdpInheritance() {
+		Udp.outboundMulticastAdapter("headers['udp_dest']")
+			.lengthCheck(true)
+			.timeToLive(10);
+	}
```

Comment:
This test doesn't look finished: no one assertion.
I understand the premise of this, but looks like a visual observation and compiler approval make a staff.
OTOH I understand also that there might be a case when we change the API and just don't have the code to prove a support for proper method chain. 
Maybe it really would be good to finish the test for the valid `MulticastSendingMessageHandler` use-case?..